shared/actions/fs/common.native.js

```diff
@@ -29,7 +29,11 @@ export const saveMedia = ({payload: {path, routePath}}: FsGen.SaveMediaPayload)
 export const pickAndUpload = ({payload: {type}}: FsGen.PickAndUploadPayload) =>
   new Promise((resolve, reject) =>
     showImagePicker(
-      {mediaType: isIOS ? 'mixed' : 'photo'}, // 'mixed' is not supported on Android. TODO: find something better.
+      {
+        mediaType: isIOS ? 'mixed' : 'photo', // 'mixed' is not supported on Android. TODO: find something better.
+        quality: 1,
+        videoQuality: 'high',
```

Comment:
Unfortunately it doesn't. On iOS there's always a "compressing video" step. I'm hoping this makes sure it's not low-quality compression. Also if we decide to make the switch to a different library, this will likely go away!